opencti-platform/opencti-front/src/private/components/Dashboard.jsx

```diff
@@ -908,25 +905,47 @@ const CustomDashboard = ({ dashboard, timeField }) => {
     </Security>
   );
 };
+
+const changeDashboardPref = graphql`
+  mutation DashboardPrefMutation($input: [EditInput]!) {
+    meEdit(input: $input) {
+      dashboard_id
+    }
+  }
+`;
+
 const Dashboard = () => {
   const classes = useStyles();
-  const [view, saveView] = useViewStorage('view-dashboard');
-  const { dashboard = 'default', timeField = 'technical' } = view;
-  const handleChangeTimeField = (event) => saveView({ dashboard, timeField: event.target.value });
-  const handleChangeDashboard = (event) => saveView({ dashboard: event.target.value, timeField });
+  const { me: { dashboard_id: dashboardId } } = useContext(UserContext);
+  const [changeDashboard] = useMutation(changeDashboardPref);
+  const [localDashboardPreferences, saveLocalDashboardPreferences] = useLocalStorage('view-dashboard', {});
```

Comment:
Ok, I now understand why we didn't understood each other yesterday. You were right, we don't need that anymore.

Reply:
There is still the date persisted (technical vs. functional) though. 

I see two possible way:
- leave it as it is now
- limit the the local storage to this value
- manage it like the dashboard (persist it on the back along with the dashboard preference)

I favor the second option, which seems to be the simplest wdyt

Reply:
Yep, let's only store the date.

Reply:
I realize that `useLocalStorage` is typed with `LocalStorage` interface, which does not seem to be designed for this kind of purpose, but to help with list pagination and various other elements manipulation.

It works as it, partly because the component is in `jsx` and partly because all fields in `useLocalStorage` are optional, but as soon as the component move to typescript, the transpiler will surely complain.

I can **create another hook**, for this special case that allow any kind of object to be stored.
Or **add another field to be handled by `LocalStorage` interface** (but that seems not a good way to resolve our problem),

do you have any other idea?